aten/src/ATen/native/native_functions.yaml

```diff
@@ -3594,6 +3594,17 @@
     MkldnnCPU: mkldnn_mul_
     NestedTensorCPU, NestedTensorCUDA: NestedTensor_mul__Tensor
 
+- func: _mul_sparse_sparse(Tensor x, Tensor y) -> Tensor
```

Comment:
Why does this need to be a new native function?

Reply:
I had issues with dispatch stubs. I.e. a code that is placed inside a `/cpu` folder is complied with different flags. Now, if the common code is moved there, linker complaints about multiple definitions. Maybe it is possible to disable AVX dispatches so that there is only a single definition...

Reply:
@nikitaved - you can leave the code outside the `cpu` folder. The `cpu` folder is really only useful if you need to take advantage of AVX2 instructions. We don't need that for now.

Reply:
So, as long as the code is placed in a cpp file the dispatcher will be able to figure out that this is a CPU kernel, right?

Reply:
OK, I was able to make CPU dispatch work, but CUDA seems less happy: `ImportError: /work/nik/pytorch/torch/lib/libtorch_cuda.so: undefined symbol: _ZN2at6native16DispatchStubImpl12get_call_ptrEN3c1010DeviceTypeEPv`.